src/tasks/FountainTask.py

```
@@ -204,10 +224,44 @@ def find_sign_in_btn(self):
             return None
         return max(regions, key=lambda region: region.width * region.height)
 
+    def click_sign_action(self, sign_btn, sign_mode):
+        self.log_info("图色识别，并确定点击选项")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)